pkg/processor/runtime/shell/runtime.go

```diff
@@ -90,8 +89,30 @@ func (s *shell) ProcessEvent(event nuclio.Event, functionLogger logger.Logger) (
 	ctx, cancel := context.WithTimeout(s.ctx, 60*time.Second)
 	defer cancel()
 
-	// create a command
-	cmd := exec.CommandContext(ctx, "sh", "-c", command)
+	var cmd *exec.Cmd
+
+	inPath, err := s.commandInPath()
```

Comment:
This should not occur on every event because the command is already known (as oppose to event details)
try ensure that before processing the event and fail on that